tests/org.pitest.pitclipse.ui.tests/old-cucumber/j06_mutations_view.feature

```diff
@@ -13,8 +13,8 @@ Feature: Mutation view shows analysis results
     Then a coverage report is generated with 2 classes tested with overall coverage of 80% and mutation coverage of 0%
     And the mutation results are
       | status      | project  | package | class       | line | mutation                                                     |
-      | SURVIVED    | project1 | foo.bar | foo.bar.Bar |    9 | negated conditional                                          |
-      | SURVIVED    | project1 | foo.bar | foo.bar.Foo |    9 | negated conditional                                          |
+      | SURVIVED    | project1 | foo.bar | foo.bar.Bar |    9 | removed conditional - replaced equality check with false     |
```

Comment:
These cucumber files are not used anymore. However, for the moment, we could keep them. I have always forgotten to remove them when I refactored the UI tests...